site/docs/patterns/split-button.mdx

```diff
@@ -0,0 +1,102 @@
+---
+title: Split button
+layout: DetailPattern
+aliases:
+  - /salt/patterns/split-button
+data:
+  components: ["Button", "Menu", "Segmented Button Group", "Tooltip"]
+---
+
+A Split Button enables users to perform a main action and have access to relevant, supplementary actions displayed inside the menu. 
+
+<Diagram
+  src="/img/patterns/split-button/split-button-hero-shot.png"
+  alt="Example showing a split button."
+  border
+/>
+
+## When to use
+
+Use the pattern when providing users with an action that has multiple commands associated with it. Users are able to immediately action the default command and access the supplementary actions, which include more specific or less commonly-used actions.
+
+For example, “Paste” functionality commands could be split as follows: 
+- Default command: Paste 
+- Alternative commands: Keep source formatting, Match formatting, Paste special...
```

Comment:
Paste special is taken from Word and ends with an ellipsis because it requires user input

Reply:
@yunjungyeh Alternative actions should be supplementary actions

Reply:
i'm not sure we need an ellipsis here as we're describing the actions rather than giving the exact microcopy. perhaps we could just use "paste text only" instead as we're missing it and it's another appropriate function.

Reply:
Sounds good, We still need to change Alternative actions to Supplementary actions to match how we refer to it in the anatomy

Reply:
```suggestion
- **Default action:** Paste.
- **Supplementary actions:** Keep source formatting, Match formatting, Paste text only.
```